### PR TITLE
docs(website): add GraphQL Serve docs

### PR DESCRIPTION
**Pull request by machi1990**

### CODE REVIEW

File: docs/graphqlserve/graphqlserve.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+$ gqlserve serve ./path/to/models --port 8080
+
+Starting server...
+
+Listening at: http://localhost:8080/graphql
+```
+
+### Enable Data Synchronization
+
+The gqlserve can also operate on data sync models. Under the hood this uses the [Data Sync](../datasync/intro.md) package.
```
Comment (craicoverflow) [inline docs/graphqlserve/graphqlserve.md:75]:
```suggestion
GraphQL Serve can also operate on data sync models. Under the hood this uses the [Data Sync](../datasync/intro.md) package. 
```

File: docs/graphqlserve/graphqlserve.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+Starting server...
+
+Listening at: http://localhost:8080/graphql
+```
+
+### Enable Data Synchronization
+
+The gqlserve can also operate on data sync models. Under the hood this uses the [Data Sync](../datasync/intro.md) package. 
+To enable data synchronization on, all we need to do is enable datasync capabilities on our models via the `@datasync` annotation.
```
Comment (craicoverflow) [inline docs/graphqlserve/graphqlserve.md:76]:
```suggestion
To enable data synchronization, all we need to do is enable datasync capabilities on our models via the `@datasync` annotation.
```

File: docs/graphqlserve/graphqlserve.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
+---
+id: graphqlserve
+title: Running an API without code
+sidebar_label: API without code
+---
+
+With GraphQL Serve you can have a full featured [GraphQL CRUD](https://graphqlcrud.org/) API with subscriptions and data synchronization running in just a few seconds without writing a single line of code - all you need is a data model `.graphql` file.
```
Comment (wtrocki) [inline docs/graphqlserve/graphqlserve.md:7]: Name with -. No strong opinion

File: docs/graphqlserve/graphqlserve.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+GraphQL Serve is a CLI tool that leverages the power of Graphback to generate a codeless Node.js GraphQL API complete with schema and CRUD resolvers and an in-memory MongoDB database.
+
+## Installation
+
+You can install `graphql-serve` globally with npm:
+
+```shell
+npm i -g graphql-serve
```
Comment (wtrocki) [inline docs/graphqlserve/graphqlserve.md:16]: For cli tools that do not have updates people recommend using npx. this way you will ensure that people using latest versions ( and we will get reliable stats about usage ;) )

File: docs/graphqlserve/graphqlserve.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+```
+
+The `@model` annotation indicates that `Note` is a data model and Graphback will generate resolvers, a CRUD service and data source for it. You can learn how to build more complex data models in [Data Model](../model/datamodel#Model).
+
+### Running your codeless GraphQL server
+
+To start your server, run the following command from the same directory as `Note.graphql`:
+
+```shell
+gqlserve serve .
```
Comment (wtrocki) [inline docs/graphqlserve/graphqlserve.md:46]: Let's add note name here. Dot can be confusing

File: docs/graphqlserve/graphqlserve.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+You can customise the directory of the data models:
+
+```shell
+gqlserve serve ./path/to/models
+```
+
+You can also specify where to load the data models from with a Glob pattern:
+
+```shell
+gqlserve print-schema ./schema/**/*.graphql
```
Comment (wtrocki) [inline docs/graphqlserve/graphqlserve.md:60]: Is that working for you?

File: docs/graphqlserve/graphqlserve.md (hunk 3 above)
Comment (machi1990) [reply docs/graphqlserve/graphqlserve.md:7]: You mean `GraphQL-Serve`?

File: docs/graphqlserve/graphqlserve.md (hunk 6 above)
Comment (machi1990) [reply docs/graphqlserve/graphqlserve.md:60]: Yes. On latest master.

File: docs/graphqlserve/graphqlserve.md (hunk 3 above)
Comment (craicoverflow) [reply docs/graphqlserve/graphqlserve.md:7]: I prefer `GraphQL Serve` not `GraphQL-Serve` when using the full name like this.

Comment (craicoverflow) [reply docs/graphqlserve/graphqlserve.md:7]: Adding the hyphen looks strange.

File: docs/graphqlserve/graphqlserve.md (hunk 4 above)
Comment (machi1990) [reply docs/graphqlserve/graphqlserve.md:16]: Do you see a section on usage with npx?

File: docs/graphqlserve/graphqlserve.md (hunk 3 above)
Comment (machi1990) [reply docs/graphqlserve/graphqlserve.md:7]: Yeap. Me too, I asked to see which name @wtrocki was referring too.

**Comment by machi1990** [comment]
Let's get this in and we can improve it with links to CRUD once the docs lands https://github.com/aerogear/graphback/pull/1655
/cc @craicoverflow

**Pull request merged**